ocaml/test/test_pvs_farm.ml

```diff
@@ -0,0 +1,54 @@
+(*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *)
+
+open OUnit
+open Test_common
+
+let name = "my_pvs_farm"
+
+let test_introduce () =
+	let __context = make_test_database () in
+	let pvs_farm = Xapi_pvs_farm.introduce ~__context ~name in
+	assert_equal name (Db.PVS_farm.get_name ~__context ~self:pvs_farm);
+	assert_equal [] (Db.PVS_farm.get_cache_storage ~__context ~self:pvs_farm)
+
+let test_forget () =
+	let __context = make_test_database () in
+	let pvs_farm = Xapi_pvs_farm.introduce ~__context ~name in
+	let pvs_proxy =
+		make_pvs_proxy ~__context ~farm:pvs_farm ~currently_attached:true () in
+	(* Forget should fail if there's a proxy. *)
```

Comment:
It should fail if there is a "running" proxy only. Let's check that it succeeds if there is a proxy that is not running.